### ISSUE TITLE
Add ability to change font size for legend text

### ISSUE THREAD

**Issue by guywithcrookedface** [masayuki0812/c3#1460]
Can you add better handling of different font sizes for legend text? From what I can tell, the calculation for position of each legend item is based off of a fixed height/width or something. When I change the font size of the ".c3-legend-item text" the different legend items overlap each-other along the X axis and the height is not recalculated to vertically align the color-chip with the vertical center of the text.

Thanks and great project!

**Comment by pavelvlasov** [comment]
+1

**Comment by aendrew** [comment]
Hi @guywithcrookedface! That is indeed a bug! Marking as such!

**Comment by guywithcrookedface** [comment]
Sweet, thanks @aendrew

**Comment by bwestpha** [comment]
Yeah, need that feature also. It would be great if i just manual can trigger to recalculate hights of the fonts etc. 

i had two pie charts with :

CSS:

.pie1 .chart .c3-legend-item {
font-size:4vh;
}

.pie2 .chart .c3-legend-item  {
fontz-size:1vh;
}

The fonts sure have different sizes, but the text-box is calculated like the initial font-size was set from the c3.css (12px) .

a fix would be great
